README.md

```diff
@@ -0,0 +1,178 @@
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/32480223/59445914-c0f5f700-8e00-11e9-8335-0b69d4728357.png"/>
+</p>
+
+# Kuzzle Plugin : Cloudinary
+
+Cloudinary is cloud-based image and video management platform. It automates the entire image-processing pipeline: from uploads to on-the-fly manipulations to optimization to dynamic delivery with any CDN.
+
+This plugin will let you use some of Cloudinary API's functions within Kuzzle. 
+
+You can test this plugin with Swagger UI. Find more in the [Testing section](#testing)
+
+***This plugin is available for 1.8.2 and higher version of Kuzzle***
+
+## Usage
+ 
+Here are the existing routes (Note that all of these are preceded by `http://<host>:<port>/_plugin/cloudinary`) 
```

Comment:
This section only describes the usage for http protocol. The related controller & action should be specifed for other protocols.